codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/config/ClientConfig.java

```diff
@@ -69,6 +67,13 @@ public String getName() {
         return name;
     }
 
+    /**
+     * @return The value of the config using the config's name.
+     */
+    public String getValue() {
```

Comment:
This method name is confusing - it does not return value.  It returns ruby code that calls the config's accessor for this config.